api/sys_init.go

```diff
@@ -58,4 +58,5 @@ type InitResponse struct {
 	RecoveryKeys    []string `json:"recovery_keys"`
 	RecoveryKeysB64 []string `json:"recovery_keys_base64"`
 	RootToken       string   `json:"root_token"`
+	AutoUnseal      bool     `json:"auto_unseal"`
```

Comment:
This isn't necessary. If what comes back is recovery keys, it was autoseal. If what comes back is barrier keys, it wasn't.

Reply:
Thanks